activestorage/lib/active_storage/engine.rb

```diff
@@ -157,5 +157,20 @@ class Engine < Rails::Engine # :nodoc:
         ActiveStorage::FixtureSet.file_fixture_path = ActiveSupport::TestCase.file_fixture_path
       end
     end
+
+    initializer "active_storage.old_signed_id_fallback" do
+      config.after_initialize do |app|
+        ActiveStorage.old_signed_id_fallback = app.config.active_storage.old_signed_id_fallback || true
```

Comment:
This will always be `true`, right? Think we can set a default value for `config.active_storage.old_signed_id_fallback` up top and do away with the `|| true`.